packages/sdk/src/linear.ts

```diff
@@ -0,0 +1,37 @@
+/* tslint:disable:no-any */
+import { GraphQLClient } from "graphql-request";
+import { issue } from "./models/issue";
+import { team } from "./models/team";
+import { label } from "./models/label";
+
+interface LinearArgs {
+  apiKey: string;
+  url?: string;
+}
+
+export class Linear {
+  public issue: ReturnType<typeof issue>;
+  public team: ReturnType<typeof team>;
+  public label: ReturnType<typeof label>;
+
+  public constructor({ apiKey, url = "https://api.linear.app/graphql" }: LinearArgs) {
+    this.client = new GraphQLClient(url, {
+      headers: {
+        Authorization: apiKey,
+        "Content-Type": "application/json",
+      },
+    });
+    this.issue = issue(this);
+    this.team = team(this);
+    this.label = label(this);
```

Comment:
Heh, this is a bit of a... simplistic mixin system. I spent a little while faffing about with the types and trying to get something more elegant, but this isn't exactly the part of the project I wanted to spent a lot of time on so this works for now.